lib/modules/apostrophe-attachments/lib/api.js

```diff
@@ -856,8 +856,23 @@ module.exports = function(self, options) {
   //
   // This method is invoked after any doc is inserted, updated, trashed
   // or rescued.
+  // We don't go further if the document has no module associated to its type
+  // (ex: pages types are not associated to modules, but they don't have attachments)
+  // Or if the document schema doesn't allow attachments.
 
   self.updateDocReferences = function(doc, callback) {
+    // Adding a 's' for plural
+    const docModule = self.apos.modules[doc.type] || self.apos.modules[`${doc.type}s`];
```

Comment:
Instead, use `self.apos.docs.getManager(doc.type)` to get the manager object. Then you can refer to the `schema` property of the manager object.

Reply:
done